README.md

```diff
@@ -34,33 +34,62 @@ Require the gem whenever you need it:
     require 'obs_github_deployments'
 ```
 
-Most of the commands require the credentials to access the GitHub repository where you are going to track the deployments. You can obtain the token in GitHub: `Settings > Developer Settings > Personal access tokens`.
+Most of the commands require the credentials to access the GitHub repository where you are going to track the deployments.
+You can obtain the token in GitHub: `Settings > Developer Settings > Personal access tokens`.
 It is enough to enable `repo:status` and `repo_deployment` scopes.
 
-You can pass them using the corresponding flags as we do in this example:
+You can pass them using the corresponding options/flags as we do in this example:
 
 ```
 obs_github_deployments unlock --repository $GITHUB_REPOSITORY --token $GITHUB_TOKEN
 ```
 
-But you could also set them as environment variables in the `.env` file taking `.env.example` as example. Doing so you don't need to specify them in the command:
+But you could also set them as environment variables in the `.env` file taking `.env.example` as an example. Doing so you don't need to specify them in the command:
 
 ```
 obs_github_deployments unlock
 ```
 
 To know the version of this gem:
 
-```
+```bash
 obs_github_deployments -v
 ```
 
-You can unlock a locked deployment at any time with:
+You might need to prevent someone to deploy in some conditions: there is a problem in the reference server, you need to wait until the maintenace window, etc.
```

Comment:
We should also lock deployments just before deploying, to prevent a second deployment while the first is in progress, right?

Reply:
I can add that reason too, yes.